demos/samv71-spi-accelerometer/src/task_set_data_output_rate.rs

```diff
@@ -0,0 +1,45 @@
+use aerugo::{logln, RuntimeApi};
+
+#[derive(Copy, Clone, Debug)]
+pub enum OutputDataRate {
+    Invalid,
```

Comment:
that doesn't make sense, as "invalid" value is not a valid value :)
Potentially invalid value should be represented as Result, not actual enum value.

Reply:
done